apps/mobile-wallet/package.json

```diff
@@ -87,6 +87,7 @@
     "react-native-background-timer": "^2.4.1",
     "react-native-gesture-handler": "^2.12.1",
     "react-native-get-random-values": "~1.8.0",
+    "react-native-localize": "^3.2.0",
```

Comment:
This package is useful to get the system language for initializing the default language